### PR TITLE
move config file handling to wallabago

### PR DESCRIPTION
**Pull request by anarcat**
Closes: #1

### CODE REVIEW

**Comment by anarcat** [comment]
obviously, this depends on https://github.com/Strubbl/wallabago/pull/3 so tests may fail right now...

File: main.go [hunk 1]
```diff
@@ -49,15 +49,10 @@ func main() {
 
 	// check for config before writing lock file to use os.Exit in case no config found
 	if *verbose {
-		log.Println("reading config")
+		log.Println("reading config", *configJSON)
 	}
-	c, err := getConfig()
+	err := wallabago.ReadConfig(*configJSON)
 	if err == nil {
```
Comment (Strubbl) [inline main.go]: The condition at this point should be err != nil

Comment (anarcat) [inline main.go]: you are totally right - fixed!

**Comment by anarcat** [comment]
not sure what's going on with the tests here:

```
./main.go:54: undefined: wallabago.ReadConfig
```

that *should* work, no?

**Comment by Strubbl** [comment]
Yes, it will work if we pull in the changes from wallabago. Do you have `govendor` in place? If yes, you can `make update-vendor` to update the vender.json file. If not, i can do that.

**Comment by anarcat** [comment]
please do, i have no idea what that is. :)

**Pull request merged**